source/crud/tests/write/findOneAndReplace-upsert.yml

```diff
@@ -0,0 +1,47 @@
+data:
+    - {_id: 1, x: 11}
+    - {_id: 2, x: 22}
+    - {_id: 3, x: 33}
+minServerVersion: '2.6'
```

Comment:
Does it make sense to leave a YML comment in this file that references SERVER-5289? It won't show up in the converted JSON of course, but it would be helpful if someone comes across this file later and wonders why it appears to duplicate the generic test.